<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,10,1">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="word_label">
          <property name="font">
           <font>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Word</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="phonetic_label">
          <property name="text">
           <string>Phonetics</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="annot_label">
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(252, 1, 7);
</string>
          </property>
          <property name="text">
           <string>Annotation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="engMean_label">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ENG MEANING</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cnMean_label">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CN MEANING</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLCDNumber" name="lcd_overall">
          <property name="styleSheet">
           <string notr="true">QLCDNumber{
	color: rgb(0, 0, 0);
	background-color: rgb(207, 207, 207)
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcd_thisTrial">
          <property name="styleSheet">
           <string notr="true">QLCDNumber{
	color: rgb(0, 0, 0);
	background-color: rgb(207, 207, 207)
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcd_correct">
          <property name="styleSheet">
           <string notr="true">QLCDNumber{
	color: rgb(100, 156, 32);
	background-color: rgb(31, 132, 59)
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcd_incorrect">
          <property name="styleSheet">
           <string notr="true">QLCDNumber{
	color: rgb(255, 0, 0);
	background-color: rgb(124, 0, 0);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="chooseButton">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="masterButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Master</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="meaningButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show Meaning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unmasterButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Unmaster</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="annot_Layout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="annot_plainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
        <property name="placeholderText">
         <string>Annotate here:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="annotButton">
        <property name="text">
         <string>Annotate</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGRE_SLAYER">
    <property name="title">
     <string>GRE SLAYER</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionEye_Sore"/>
    <addaction name="actionFile_Directory"/>
    <addaction name="actionAnnotate"/>
    <addaction name="actionDictionary"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionMeaning"/>
    <addaction name="actionNew_Words_Only"/>
    <addaction name="actionReview_Mode"/>
    <addaction name="actionTime_Machine"/>
   </widget>
   <addaction name="menuGRE_SLAYER"/>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMeaning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Meaning</string>
   </property>
  </action>
  <action name="actionNew_Words_Only">
   <property name="text">
    <string>New Words Only</string>
   </property>
  </action>
  <action name="actionReview_Mode">
   <property name="text">
    <string>Review Mode</string>
   </property>
  </action>
  <action name="actionTime_Machine">
   <property name="text">
    <string>Time Machine</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionEye_Sore">
   <property name="text">
    <string>Sored Eye?</string>
   </property>
  </action>
  <action name="actionFile_Directory">
   <property name="text">
    <string>File Directory</string>
   </property>
  </action>
  <action name="actionAnnotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Annotate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionDictionary">
   <property name="text">
    <string>Dictionary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
